mlpp_lib/normalizers.py

```
@@ -69,7 +67,7 @@ def fit(self, dataset: xr.Dataset, dims: Optional[list] = None):
 
         for name, (transformer, variables) in self.transformers.items():
             try:
-                transformer.fit(dataset=dataset, variables=variables, dims=dims)
+                transformer.fit(dataset=dataset[variables], dims=dims)
                 LOGGER.info(f"Fitted {name} to variables: {variables}")
             except Exception as e:
                 LOGGER.error(f"Failed to fit {name}: {e}")
```

Now that I see that, should we also raise the error ?